common/dpsl.ts

```diff
@@ -213,10 +364,14 @@ export type DpslTypes =
   | StatefulPartitionInfo
```

Comment:
Is it expected that there is only one "DpslTypes" here?

Reply:
It is not related to the Diagnostics service.